CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # git-dit ChangeLog
 
+## v0.4.0 (2017-09-15)
+
+### Binary
+
+Changes:
+ * The message formatting for the "show" subcommand was improved.
+ * In filter-specs for the "list" subcommand, a '=' is used for separating key
+   and value when filtering for an exact value, rather than a ':'.
+
+Added features:
+ * The "create-message", "new", "reply" and "tag" now accept the "--author" and
+   "--date" options. Analogue to the options of "git commit" with the same name,
+   the new options alter the author and authoring date of the resulting message.
+ * New "reporter-name" and "reporter-email" filter keys for issue filtering
+   using the "list" subcommand. As the name suggests, the keys may be used for
+   filtering issues based on the reporter's name or email.
+ * Filter specs for the "list" subcommand can now be used for filtering by
+   substring or presence of a piece of issue metadata.
+ * Filter specs for the "list" subcommand can now be inverted.
+
+Bug-fixes:
+ * Git-dit did not handle pager and editor flags correctly. For example,
+   configuring `less -R` as the pager (contrary to `less`) would cause git-dit
+   search for a binary named "less -R", resulting in an error.
+
+### Library
+
+Changes:
```

Comment:
Are these changes backwards-compatible? As I think they are mostly not, I would state that. Of course we are pre-1.0, but it doesn't hurt to say "Backwards-incompatible Changes" here rather than not specifying exactly what this list is about.

Reply:
I consider points in "Changes" sections generally backwords-incompatible. And we used the term in past releases.